class.jetpack.php

```diff
@@ -1249,6 +1264,31 @@ public static function is_version_controlled() {
 		}
 		return $is_version_controlled;
 	}
+
+	/**
+	 * Retrieves a list of allowed file types.
+	 * @return array $allowed_file_types
+	 **/
+	public static function allowed_file_types() {
+		$allowed_file_types= array();
+		$mime_types = get_allowed_mime_types();
```

Comment:
That's an interesting point. Based on that, I'm not sure that syncing this value as is would be any more accurate than what we have now.

Has there been discussion before about syncing user specific information?